README.md

```diff
@@ -0,0 +1,13 @@
+# PROJECT: RESTAURANT PAGE
+The aim of this project was to implement javascript ES6 import/export module pattern by using webpack. You can see the full instructions on [the odin project](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page).
+
+**Technology Used**
+* Javascript
+* HTML
+* CSS
+* Eslint
+* webpack
+
+**Authors**
+### [Nyaga Roy](https://github.com/RoyNyaga).
+
```

Comment:
We can make the readme more descriptive by adding more details about the project like screenshot and live link, eg: https://raw.githack.com/RoyNyaga/microverse_restaurant_page_js/f6d9d2e69612ad2d78dc3aea398b7f7bccbf2248/dist/index.html.

Reply:
Ok